pages/serializers.py

```diff
@@ -185,10 +185,10 @@ class PageElementSerializer(serializers.ModelSerializer):
         help_text=_("Account that can edit the page element"))
     picture = serializers.CharField(required=False, allow_null=True,
         help_text=_("Picture icon that can be displayed alongside the title"))
-    text = HTMLField(required=False,
-        html_tags=settings.ALLOWED_TAGS,
-        html_attributes=settings.ALLOWED_ATTRIBUTES,
-        help_text=_("Long description of the page element"))
+    content_format = serializers.ChoiceField(
```

Comment:
I am wondering if it is possible to guess the `content_format` from the `text` itself. Maybe not, but that would be awesome.